src/usr/local/www/firewall_nat_1to1_edit.php

```diff
@@ -461,7 +461,7 @@ function dsttype_selected() {
 	'interface',
 	'*Interface',
 	$pconfig['interface'],
-	$interfaces
```

Comment:
You should be able to remove the code that builds `$interfaces`, or replace it with `$interfaces = build_interface_list();` and then leave this part as `$interfaces`, depending on what else uses $interfaces.